### PR TITLE
Add definition for redux-ui

### PR DESCRIPTION
**Pull request by blaesus**
case 1. Add a new type definition.
- [X] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [X] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

`redux-ui` API documentation: https://github.com/tonyhb/redux-ui

### CODE REVIEW

File: redux-ui/redux-ui.d.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/// <reference path="../redux/redux.d.ts"/>
+
+declare module "redux-ui" {
+
+  import * as Redux from 'redux'
```
Comment (zhengbli) [inline redux-ui/redux-ui.d.ts]: Semicolon at the end of each line

**Comment by andyshuxin** [comment]
@zhengbli semicolons added

File: redux-ui/redux-ui.d.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/// <reference path="../redux/redux.d.ts"/>
```
Comment (Diullei) [inline redux-ui/redux-ui.d.ts]: the header content is missing. see http://definitelytyped.org/guides/contributing.html#content

Comment (andyshuxin) [inline redux-ui/redux-ui.d.ts]: @Diullei updated, thanks.

**Comment by zhengbli** [comment]
👍

**Pull request merged**